qiskit_experiments/curve_analysis/fit_function.py

```diff
@@ -45,6 +46,7 @@ def _wrapper(x, *args, **kwargs) -> Union[float, UFloat, np.ndarray]:
     return _wrapper
 
 
+@deprecated_function("0.5", "Now fit function can be defined with Python string.")
```

Comment:
I would argue you could keep these functions and use them as `Model` funcs rather than using `ExpressionModels`, especially if the same fit function is used for multiple experiments. For one off functions maybe an expression model is fine.

Reply:
Removed deprecation from several functions. Once dedicated to CR Hamtomo fitting is still under deprecation. Note that many of these functions are not used as-is as you expected, because each analysis wants to define own parameter name by signature replacement with lambda. I think calling these functions with lambda doesn't improve readability, and use of raw string is more readable.

I agree user can still use these functions if they are happy with existing signature. Note that if you pass function to LMFIT model, it calls `inspect` and parses its signature. So something is parsed anyways.

Reply:
Reading LMFit docs it sounds like you could use the `indepdendent_vars` kwarg of `lmfit.Model` to specify the params of the general function you don't want to fit rather than using a lambda to wrap it. But I think then you need to supply the fixed value later for calling fit, which might not be convenient in the current setup